11.Portret.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	<script>
+		const year = prompt("Напишіть свій рік народження");
+		const city = prompt("В якому місті ви живете: Київ, Вашингтон, Лондон або в іншому?");
+		const sports = prompt("Ваш улюлений вид спорту футбол, баскетбол або бокс?");
+
+		const Result = () => {
```

Comment:
In javascript, we use function name with first uppercase letter for naming constructors

Rename, plz `Result` -> `result`